### ISSUE TITLE
Attachments? Images?

### ISSUE THREAD

**Issue by mspangdal** [deltaprojects/waffleio#2]
Issuetracker without image upload support tend to be less than ideal

**Comment by CarlRosell** [comment]
That's true. And I think they removed the feature to move issues between projects. It destroyed some peoples changelogs.
Perhaps having a standalone solution would be best.
